static/app/views/organizationStats/usageStatsOrg.tsx

```diff
@@ -39,17 +41,26 @@ type Props = {
     pagePeriod?: string | null;
     transform?: ChartDataTransform;
   }) => void;
+  isSingleProject: boolean;
   organization: Organization;
   projectIds: number[];
   chartTransform?: string;
 } & AsyncComponent['props'];
 
-type State = {
+type UsageStatsOrganizationState = {
   orgStats: UsageSeries | undefined;
 } & AsyncComponent['state'];
 
-class UsageStatsOrganization extends AsyncComponent<Props, State> {
-  componentDidUpdate(prevProps: Props) {
+/**
+ * This component is replaced by EnhancedUsageStatsOrganization in getsentry, which inherits
```

Comment:
given this note, do we have any tests for this component?

Reply:
Yep, this component has test coverage :+1: